Cargo.toml

```diff
@@ -1,6 +1,2 @@
 [workspace]
 members = ["rust/crd", "rust/operator-binary"]
-
-# [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { path = "/home/sbernauer/stackabletech/operator-rs" }
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "feat/affinities" }
```

Comment:
I would prefer to leave this in, to make it easier to quickly use a local version

Reply:
Re-added in 87a973d